src/cmd/services/m3coordinator/ingest/write.go

```diff
@@ -124,13 +126,26 @@ func NewDownsamplerAndWriter(
 	workerPool xsync.PooledWorkerPool,
 	instrumentOpts instrument.Options,
 ) DownsamplerAndWriter {
-	scope := instrumentOpts.MetricsScope().SubScope("downsampler")
+	var (
+		scope = instrumentOpts.MetricsScope().SubScope("downsampler")
+
+		emptyAnnotationMetricName = "datapoints_with_empty_annotation"
```

Comment:
Nit: maybe let's shorten this to 
```suggestion
		emptyAnnotationMetricName = "writes_without_annotation"
```
(and all the related symbols, accordingly)

Reply:
https://github.com/m3db/m3/pull/3553/commits/80e0205445535c5b780dcf63e623efc65cf98e27#diff-bd39ac14b90a7e77e48b860a55e1927d98fd3378a61676da3b11c8b4a7698958R132

https://github.com/m3db/m3/pull/3553/commits/f0e4b8a8b5cd6a90883a27d66efd5bb42fac9490

https://github.com/m3db/m3/pull/3553/commits/b3fe075280c69f5393d160a1e272a909f690c5b1